demos/01-create-app/graph-tutorial/package.json

```diff
@@ -11,42 +11,42 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^7.2.7",
-    "@angular/common": "^7.2.7",
-    "@angular/compiler": "^7.2.7",
-    "@angular/core": "^7.2.7",
-    "@angular/forms": "^7.2.7",
-    "@angular/platform-browser": "^7.2.7",
-    "@angular/platform-browser-dynamic": "^7.2.7",
-    "@angular/router": "^7.2.7",
+    "@angular/animations": "^7.2.14",
```

Comment:
Please update create-app.md with new versions

Reply:
Fixed in most recent commit... missed that in the commit/push.